packages/core/backend/sirius-components-representations/src/main/java/org/eclipse/sirius/components/representations/VariableManager.java

```diff
@@ -49,6 +49,11 @@ public class VariableManager {
      */
     public static final String SELF = "self";
 
+    /**
+     * The name of the variable used to store the ancestors (all previous self) of the current context.
+     */
+    public static final String ANCESTORS = "ancestors";
+
```

Comment:
This is highly specific to diagrams and thus has nothing to do in VariableManager. This could be in NodeDescription for example.